public/app/features/dashboard/components/PanelEditor/VisualizationSelectPane.tsx

```diff
@@ -74,7 +74,12 @@ export const VisualizationSelectPane: FC<Props> = ({ panel }) => {
 
   const radioOptions: Array<SelectableValue<ListMode>> = [
     { label: 'Visualizations', value: ListMode.Visualizations },
-    { label: 'Global panels', value: ListMode.Globals },
+    {
+      label: 'Global panels',
+      value: ListMode.Globals,
+      icon: 'info-circle',
+      description: 'Reusable panels whose state is reflected on each dashboard they are used with.',
```

Comment:
I'm not a copy writer ;) but what about this?
```suggestion
      description: 'Panels you can share between dashboards.',
```

Reply:
This is much better haha, will change